install.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+org="puppetlabs"
+repo="pdkgo"
+
+app="pct"
+
+arch="x86_64"
+# os="$(uname -s | tr '[:upper:]' '[:lower:]')"
```

Comment:
One question @jpogran - I noticed [archives created in your prototype repo ](https://github.com/jpogran/prototype/releases/tag/v0.1.0) had uppercase names for the OS. `goreleaser` does not seem to capitalise on my system. Just curious how these were generated?

Reply:
For this repo I customized the goreleaser archive config on line [32](https://github.com/puppetlabs/pdkgo/blob/main/.goreleaser.yml#L32) with `toLower` on the OS name to keep all parts the same case.